### PR TITLE
Robot Framework client update for version 3.0.0

### PR DESCRIPTION
**Pull request by krasoffski**

### CODE REVIEW

File: robotframework_reportportal/model.py [hunk 1]
```diff
@@ -42,33 +39,38 @@ def __init__(self, name=None, attributes=None):
...
 
-    def get_kwd(self):
-        if self.assign:
-            return "{0}{1}.{2}{3}".format(
-                "{0} = ".format(get_list_as_str(self.assign)),
-                self.libname, self.keyword_name,
-                "({0})".format(
-                    get_list_as_str(self.args)))
+    def get_name(self):
+        return "{0}{1} ({2})".format(
```
Comment (krasoffski) [inline robotframework_reportportal/model.py]: Could you rewrite this function in more `pythonic` way?

File: robotframework_reportportal/service.py [hunk 2]
```diff
@@ -27,207 +30,134 @@ class RobotService(object):
...
         if RobotService.rp is None:
             logging.debug(
-                msg="ReportPortal - Init service: "
-                    "endpoint={0}, project={1}, uuid={2}".format(endpoint,
-                                                                 project,
-                                                                 uuid))
-            RobotService.rp = ReportPortalService(endpoint=endpoint,
-                                                  project=project,
-                                                  token=uuid)
+                "ReportPortal - Init service: "
```
Comment (krasoffski) [inline robotframework_reportportal/service.py]: Please, walk through all debug messages and use one log style: `canonical` like bellow or using `str.format`.

File: setup.py [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 ]
 
 setup(
     name='robotframework-reportportal',
     packages=find_packages(),
-    version='2.5.5',
+    version='3.0.0',
     description='Listener for RobotFramework reporting to ReportPortal',
     author='Artsiom Tkachou',
     author_email='artsiom_tkachou@epam.com',
```
Comment (krasoffski) [inline setup.py]: Update email with RP main like: `SupportEPMC-TSTReportPortal@epam.com`

File: robotframework_reportportal/model.py (hunk 1 above)
Comment (vakulich) [inline robotframework_reportportal/model.py]: fixed

File: robotframework_reportportal/service.py (hunk 2 above)
Comment (vakulich) [inline robotframework_reportportal/service.py]: fixed

File: setup.py (hunk 3 above)
Comment (vakulich) [inline setup.py]: updated

**Pull request merged**